### PR TITLE
Discuss: One does not simply

### PR DESCRIPTION
**Pull request by johanandren**
There was a tweet which I cannot find that expressed how bad it is to have tech docs say things like "simply do this", "you can easily do that", "obviously it is like this".

I think there was more but cannot re-find the tweet, discussed this with @chbatey a few days ago, maybe he knows. 

WDYT?

### CODE REVIEW

**Review by raboof — APPROVED** [review]
I generally like the direction and the changes here. Sometimes 'easily' can be replaced by 'conveniently' - if it's not considered convenient then at least that's clearly our fault and we're not insulting the readers' intelligence.

**Review by ktoso — APPROVED** [review]
LGTM, good change.

**Comment by johanandren** [comment]
Wait with merging until Chris (in his polite British way) can tell me what more words there is that insult the reader's intelligence and I'll fix those as well.

**Comment by raboof** [comment]
https://twitter.com/jesstelford/status/992756386160234497 may have been the one you saw - perhaps @rstento may have some good guidance here as well?

**Comment by rstento** [comment]
Only had time to take a quick look, @raboof, but the removal of "easily" and "simply" simply made these instructions better for the reader to follow more easily. : -)

**Comment by johanandren** [comment]
Skipping "however", "so" and "easy" because I do not quite agree with.

**Review by chbatey — APPROVED** [review]
Looks great

File: akka-docs/src/main/paradox/actors.md [hunk 1]
```diff
@@ -568,7 +568,7 @@ sending their ActorRefs to other Actors within messages.
 
 @@@
 
-The supplied path is parsed as a `java.net.URI`, which basically means
```
Comment (ktoso) [inline akka-docs/src/main/paradox/actors.md]: my favourite word of all time :-) Anyway, good to remove :)

**Comment by ktoso** [comment]
LGTM again ;-)

**Review by patriknw — COMMENTED** [review]
good stuff!

**Pull request merged**